### ISSUE TITLE
Comhairle accessibility audit

### ISSUE THREAD

**Issue by shuyanglin** [crownshy/comhairle#42]
I have this docuemnt collecting links and information around accessibility 
https://docs.google.com/document/d/1HmNOA7OxKhQHVPxxHqzUtM-h6NFtu4tGUHRMZcI0NaI/edit?tab=t.0#heading=h.3g159h6ht7kk
Also added a few questions I have just now, maybe we can jump on a call for just 10 minutes of your time to discuss? 

Two tools that should be useful

automatic WCAG level AA detection tool - it can be integrated in the CI/CD pipeline and catch basic accessibility issues like color contrast, missing alt-text
https://github.com/dequelabs/axe-core

This is voice over tool that many use
https://webaim.org/articles/voiceover/

There are also other tools like WAVE (this name jumped out often but I haven't investigate this fully)  https://wave.webaim.org/

**Comment by dtodorowa** [comment]
The MyWeb Access baseline audit (v1.2) is now broken out into a remediation epic with sub-issues: #688. The full report PDF still needs attaching (to #688 or the Google Doc above) so the finding IDs resolve.
